### DISCUSSION TITLE
AppImages

### DISCUSSION THREAD

**Discussion by Lonniebiz** [VSCodium/vscodium#1741] [General]
First of all, thank you for supporting AppImages. I notice that some releases don't have AppImages for download.

For example, while the latest stable release (as I type this) seems to be [1.84.2.23319](https://github.com/VSCodium/vscodium/releases/tag/1.84.2.23319), the latest AppImage I see is version [1.81.0.23216](https://github.com/VSCodium/vscodium/releases/download/1.81.0.23216/VSCodium-1.81.0.23216.glibc2.17-x86_64.AppImage).

I'm glad to have 1.81, to be honest, but was curious why there's no AppImage for 1.84. Is making the AppImage part of a grand build process, or does it require a lot of extra effort for you each release. Either way, thank you so much for being so accommodating.

**Comment by Smojo** [comment]
Seems like this ~~[1.82.0.23250](https://github.com/VSCodium/vscodium/releases/tag/1.82.0.23250)~~ - correction: ([1.87.2.24072](https://github.com/VSCodium/vscodium/releases/tag/1.87.2.24072))
was the latest, but still, it seems like it will not be provided for any release.

At least it is still listed under:
https://github.com/VSCodium/vscodium?tab=readme-ov-file#supported-platforms 
GNU/Linux x64 (deb, rpm, **AppImage**, snap, tar.gz)
